pkgs/tools/filesystems/netatalk/default.nix

```diff
@@ -4,12 +4,10 @@
 
 stdenv.mkDerivation rec {
   pname = "netatalk";
-  release = "3.1.13";
-  patch = "3";
-  version = "${release}_${patch}";
+  version = "3.1.13";
```

Comment:
What's the idea behind going from `3.1.13_3` to `3.1.13`? 3.1.13 is the upstream release and the package is patched quite a bit.

Reply:
nix does not rely on version numbers to compare packages so we do not need to modify them and introduce artificial ones.

Reply:
Thanks for clarifying. I was not able to find versioning guidance in the Nix docs (perhaps wasn't looking hard enough).

Reply:
The version guidance is basically to use what upstream has and strip prefixes like v and RELEASE_